lutris.spec

```diff
@@ -51,12 +51,14 @@ BuildRequires: fdupes
 %ifarch x86_64
 Requires:       mesa-vulkan-drivers(x86-32)
 Requires:       vulkan-loader(x86-32)
+Requires:       mesa-libGL(x86-32)
 %endif
 
 Requires:       mesa-vulkan-drivers
 Requires:       vulkan-loader
 Recommends:     wine-core
 BuildRequires:  fdupes
+Requires:       mesa-libGL
```

Comment:
Ditto. Should be added under `%if 0%{?fedora}` instead (along with `mesa-dri-drivers`).